### DISCUSSION TITLE
Manage Individual Movie Posters

### DISCUSSION THREAD

**Discussion by userspaced** [meisnate12/Plex-Meta-Manager#299] [Q&A] [answered]
I was wondering if I could manage the posters for individual movies within a collection. I was trying to figure out of this is possible and started down the path of trying to use `tmdb_movie_details` but couldn't really figure out how to do it. Is it possible?

**Comment by meisnate12** [comment, marked as answer]
theres two ways you can do that either

1. You can create a metadata entry for the movie/show you want to edit and use an [Image Attribute](https://github.com/meisnate12/Plex-Meta-Manager/wiki/Metadata-Attributes#image-attributes) to edit the poster or background.
2. You can use the [Asset's Directory](https://github.com/meisnate12/Plex-Meta-Manager/wiki/Settings-Attributes#image-asset-directory) to update posters of collections, movies, and shows.
